openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -327,6 +327,11 @@ def _tile_render(self, payload):
         job_info = copy.deepcopy(payload_job_info)
         plugin_info = copy.deepcopy(payload_plugin_info)
 
+        # Force plugin reload for vray cause the region does not get flushed
+        # between tile renders.
+        if plugin_info["Renderer"] == "vray":
+            job_info.ForceReloadPlugin = True
```

Comment:
This really sounds more like a Deadline Plugin issue which might be better solved there. Anyway, interesting catch!

Reply:
Yup, I agree but we are using built-in Draft Tile Assembler so we are at the hands of Thinkbox.